realm/realm-library/src/androidTest/java/io/realm/OrderedRealmCollectionTests.java

```diff
@@ -221,6 +221,14 @@ public void first_emptyCollection() {
     }
 
     @Test
+    public void first_withDefault() {
+        AllJavaTypes defaultObject = collection.get(0);
```

Comment:
In this case just doing `new AllJavaTypes()` should work I think? Since the default value is not manipulated in any way?